style.css

```diff
@@ -120,6 +120,28 @@ nav li {
     border: #ffffff;
 }
 
+.me {
+    width: 80%;
+    margin: 0 auto;
+    padding-top: 60px;
```

Comment:
this is to stop the white band appearing im guessing, rather than doing this just set the margin top of the h1 to 0

Reply:
just to move the whole dive down from the top